capitalize.decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'DecoratorBase'
+
+class CapitalizeDecorator < DecoratorBase
```

Comment:
- [ ] It would be nice if you fix the naming of the files and classes here because there is file decorator.rb but you have imported DecoratorBase which is incorrect and giving me an error 😨 Kindly import decorator 👍
- [ ] It would be nice if you use the correct Parent class name because you have a class Decorator and not DecoratorBase 😨 Kindly fix it to make your app error-free 😊


After making changes it would look like the below ⬇️

```ruby
require_relative 'decorator'

class TrimmerDecorator < Decorator
```

Reply:
@HAMMAS-SALEEM Done

Reply:
- Well resolved